ai/priority_algorithm/priority_algorithm.py

```diff
@@ -20,8 +20,8 @@ class PriorityAlgorithm(WeightAlgorithm):
 
     MAX_KEEP: int = 3  # nodes
     MAX_SPREAD: int = 3  # nodes
-    MAX_ITERATE: int = 15  # times
-    MAX_TIME: int = 30  # seconds
+    MAX_ITERATE: int = 1500  # times
+    MAX_TIME: int = 7  # seconds
```

Comment:
Are these just arbitrary numbers or is there a reason you choose these?

Reply:
Arbitrary, might have gotten a bit ambitious given the local dev experience